webui/react/src/services/decoder.ts

```diff
@@ -528,32 +528,36 @@ const decodeV1LogLevelToLogLevel = (level: Sdk.V1LogLevel): types.LogLevel | und
 const defaultRegex = /^\[([^\]]+)\]\s([\s\S]*)(\r|\n)$/im;
 const kubernetesRegex = /^\s*([0-9a-f]+)\s+(\[[^\]]+\])\s\|\|\s(\S+)\s([\s\S]*)(\r|\n)$/im;
 
+const filterLogMessage = (message: string): string => {
+  let filteredMessage = message.trim();
+
+  if (defaultRegex.test(message)) {
```

Comment:
I'm not sure how .test is implemented but if it's just checking for matches underthe hood just doing `.match` once and storing the results should be faster considering we might be processing a lot of log lines during scroll events.

Reply:
According to [this website](https://www.measurethat.net/Benchmarks/Show/11043/0/stringmatch-vs-regextest), `.test` is about 58% faster than `.match`.

Reply:
good call